fdbserver/BlobWorker.actor.cpp

```diff
@@ -3046,6 +3195,14 @@ ACTOR Future<Void> doBlobGranuleFileRequest(Reference<BlobWorkerData> bwData, Bl
 					}
 				}
 
+				if (chunk.snapshotFile.present() && chunk.snapshotFile.get().cipherKeysMetaRef.present()) {
```

Comment:
Doesn't have to be for this PR, but in the future we should consider pipelining these lookups instead of waiting on them, if there are multiple granules in the read response. We should at least add a TODO here.
We also will want to have latency information for the granule keys lookups, so we can factor that in to the latency of these requests

Reply:
Great idea!
Updated the code to allow EK lookup parallelize with mutations hydration to the response. 
Thanks